inlong-sort/sort-connectors/elasticsearch-base/src/main/java/org/apache/inlong/sort/elasticsearch/table/RowElasticsearchSinkFunction.java

```diff
@@ -119,6 +148,7 @@ private void processUpsert(RowData row, RequestIndexer indexer) {
         if (sinkMetricData.getNumBytesOut() != null) {
             sinkMetricData.getNumBytesOut().inc(document.length);
         }
+        outputMetricForAudit(document.length);
```

Comment:
Here only output +I/+U data？

Reply:
@thesumery should  the rowData of DELETE mode be counted?